### PR TITLE
Create 2016-05-12-TGIF-TBD.md

### PR DESCRIPTION
**Pull request by skeskali**

### CODE REVIEW

**Comment by jennwrites** [comment]
Looks great, thanks so much @skeskali! LGTM

File: _posts/2016-05-12-beer-card-pet-projects-tgif-83.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+[Why Designers Write on the Walls (And Why You Should Too)](https://medium.com/@george_aye/always-show-your-work-why-designers-write-on-the-walls-and-why-you-should-too-6725237b9765?mc_cid=c3898e3176&mc_eid=549abb3da2#.6ie4rkobz)
+
+[Resource Cards](https://resourcecards.com/?mc_cid=c3898e3176&mc_eid=549abb3da2) is a list of free resources for creatives
+
+
+## The Web and Development
+
+[Locking In Webkit](https://webkit.org/blog/6161/locking-in-webkit/)
+
```
Comment (varjmes) [inline _posts/2016-05-12-beer-card-pet-projects-tgif-83.md:47]: how comes there are double newlines here but single newlines in other sections?

**Comment by lewiscowper** [comment]
LGTM too! 👍

File: _posts/2016-05-12-beer-card-pet-projects-tgif-83.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+[Tradeoffs In Coordination Among Teams](http://blog.jessitron.com/2016/05/tradeoffs-in-coordination-among-teams.html)
+
+
+[Here’s a Slack slash command to let your teams know when a task is done.
+](https://hoodie-community.slack.com/apps/A0PB7H5AT--done)
```
Comment (varjmes) [inline _posts/2016-05-12-beer-card-pet-projects-tgif-83.md]: github isn't showing that this has been formed into a proper URL, is someone able to check how it renders?

**Comment by gr2m** [comment]
Added shoutout to Stephanie’s new tune :)

File: _posts/2016-05-12-beer-card-pet-projects-tgif-83.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
+
```
Comment (gr2m) [inline _posts/2016-05-12-beer-card-pet-projects-tgif-83.md]: must not have an empty line in the beginning. Just an FYI for the future @skeskali, no biggie, I’ll fix it :)

**Comment by gr2m** [comment]
LGTM 👍 I updated a few minors, see https://github.com/hoodiehq/hood.ie/pull/286/commits. I checked it locally, it’s all looking good. :shipit:

**Comment by jennwrites** [comment]
Alright let's publish!

**Pull request merged**